lib/mirage.ml

```diff
@@ -443,7 +443,9 @@ let archive_conf = impl @@ object
     method name = "archive"
     method module_name = "Tar_mirage.Make_KV_RO"
     method packages =
-      Key.pure [ package ~min:"0.7.0" ~ocamlfind:["tar.mirage"] "tar-format" ]
+      Key.pure [
+        package ~min:"0.7.0" ~ocamlfind:["tar-format.mirage"] "tar-format"
+      ]
```

Comment:
That's correct -- from version 0.8.0 `tar-format` -> `tar` and `tar-format.mirage` -> `tar-mirage`. The old packages should still work but you might as well update.